src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
+
+* [Event gateways with Kafka](/hub/kong-inc/kafka-upstream/)
+* [GraphQL](/hub/kong-inc/graphql-proxy-cache-advanced/)
+* [Mocking](/hub/kong-inc/mocking/)
+* [Advanced data transformation](/hub/kong-inc/jq/)
+* [OPA Policy driven traffic management](/hub/kong-inc/opa/)
+* [API product tiers](/gateway/{{page.kong_version}}/admin-api/consumer-groups/reference/#main)
+
+[Get started plugins &rarr;](/hub/)
+
+## Dev Portal
+
+The Dev Portal provides a single source of truth for all developers to locate, access and consume APIs, similar to a traditional API Catalogue. [Dev Portal](/gateway/{{page.kong_version}}/kong-enterprise/dev-portal/) streamlines developer onboarding by offering a self-service developer experience to discover, register, and consume published services from the Service Hub catalog. This customizable experience can be used to match your own unique branding and highlights the documentation and interactive API specifications of your services. Enable application registration to automatically secure your APIs with a variety of authorization providers.
+
+[Learn more about Dev Portal &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/dev-portal/)
 
 ## Monitoring and analytics
 
-Use [Vitals](/gateway/{{page.kong_version}}/kong-enterprise/analytics/) to gain deep insights into service, route, and application usage and health monitoring data. Keep your finger on the pulse of the health of your API products with custom reports and contextual dashboards. In addition, you can enhance the native monitoring and analytics capabilities with {{site.base_gateway}} plugins that enable streaming monitoring metrics to third-party analytics providers, such as Datadog and Prometheus.
+The [Analytics](/gateway/{{page.kong_version}}/kong-enterprise/analytics/) platform gives you deep insights into services, routes, and application usage data. You can keep your finger on the pulse of the health of your API products with custom reports and contextual dashboards, and you can enhance the native monitoring and analytics capabilities with {{site.base_gateway}} plugins that enable streaming monitoring metrics to third-party analytics providers, such as [Datadog](/hub/kong-inc/datadog/) and [Prometheus](/hub/kong-inc/prometheus/).
 
 [Start monitoring with Vitals &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/analytics/)
 
+## Role-based access control (RBAC)
+
+Kong Enterprise allows the use of built-in RBAC so you can streamline developer onboarding, and create apply fine-grained security and traffic policies using the Admin API, Kong Manager, and the CLI.
+You can configure {{site.base_gateway}} users, roles, and permissions with role-based access control (RBAC), either through the [Admin API](/gateway/{{page.kong_version}}/admin-api/rbac/reference/) or with [Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/rbac).
```

Comment:
These two sentences say the same thing in slightly different ways. 
```suggestion
Kong Enterprise lets you configure users, roles, and permissions with built-in role-based access control (RBAC). With RBAC, you can streamline developer onboarding, and create apply fine-grained security and traffic policies using the [Admin API](/gateway/{{page.kong_version}}/admin-api/rbac/reference/), [Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/rbac), and the CLI.
```
I don't think the CLI is correct. The kong CLI doesn't have any ability to manage RBAC.
https://docs.konghq.com/gateway/latest/reference/cli/
Neither does the Dev Portal one:
https://docs.konghq.com/gateway/latest/kong-enterprise/dev-portal/cli/

So unless there's a CLI we don't know about...